CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- feat(android): Add wizard support for Android (#389)
```

Comment:
l: When adding a new wizard snippet, we usually show how to invoke the wizard in the changelog (e.g. see entry for version 3.2.0)